tools/mktutorial/main.go

```diff
@@ -169,11 +169,13 @@ func gatherTutorials(root string) ([]tutorial, error) {
 			continue
 		}
 
-		// Add the language to the page title to avoid duplicate titles.
+		// Add the language to the page title and meta descriptions to avoid duplicates.
 		var title string
+		var metaDescription string
 
 		if val, ok := langMap[language]; ok {
 			title = fmt.Sprintf("%s | %s", h1, val)
+			metaDescription = fmt.Sprintf("%s How-to Guide using %s", h1, val)
```

Comment:
What do y'all think of also appending the like the first 70 characters of the body to this message as well. The generated descriptions look a bit short, so just trying to think of a quick way to get some more length on them.

Reply:
i think this is a good idea, but would like to do it in a separate pr